PZ/WebClient/models.py

```diff
@@ -6,3 +6,140 @@ class Monitors(models.Model):
     monitor_name = models.CharField(max_length=200)
     monitor_domain = models.URLField()
     user_id = models.ForeignKey(User)
+
+
+class Page(object):
+	def __init__(self, size, number, totalCount):
+		# self.dict = json.loads(json_obj)
+		self.size = size
+		self.number = number
+		self.totalCount = totalCount
+
+	def set_size(self, size):

```

Comment:
Use properies instead of get / set methods
```@property
def size(self):
    return self._size

@size.setter
def size(self, value):
    self._size = value
```
With it, you can make needed variables set-only or get-only, or add some constraints